seqr/views/utils/test_utils.py

```diff
@@ -205,9 +216,17 @@ def get_initial_page_window(self, key, response):
     def get_initial_page_json(self, response):
         return self.get_initial_page_window('initialJSON', response)
 
+    def check_no_analyst_no_access(self, url, get_response=None):
```

Comment:
The `url` can be optional.

Reply:
I'd rather not, it makes it harder to read/maintain and its best practice to include it